xapian-applications/omega/omegatest

```diff
@@ -655,6 +655,13 @@ testcase 'b4c216e4da73d1d01277ef46d0514821' text="simple query"
 testcase 'd41d8cd98f00b204e9800998ecf8427e' text=""
 testcase '3302c94d3500c3f695b7e09b7acd420d' text=" test "
 
+# Feature tests for random{}
+printf '$random{$cgi{x},$cgi{y}}' > "$TEST_TEMPLATE"
+testcase '89' x=89 y=90
+
+printf '$random{$cgi{x}}' > "$TEST_TEMPLATE"
+testcase '0' x=1
+
```

Comment:
This should really test that the random number returned is within the expected range for some non-degenerate cases - for example, generate 100 numbers with `$random{6}` and check each is both >= 0 and < 6.

(And it doesn't test `$random` without arguments at all, but I think that should be removed as I say above).

Reply:
I wrote a new function for checking rng testcases inside the script.